app/assets/javascripts/discourse/app/static/prosemirror/extensions/trailing-paragraph.js

```diff
@@ -0,0 +1,47 @@
+/** @type {RichEditorExtension} */
+const extension = {
+  plugins({ pmState: { Plugin, PluginKey } }) {
+    const plugin = new PluginKey("trailing-paragraph");
```

Comment:
Should we be using this in all the extensions? Seems like it would be useful:

![image](https://github.com/user-attachments/assets/3eb8d1c3-4534-4abd-9349-f07e1e6fd50c)

Reply:
https://prosemirror.net/docs/ref/#state.PluginSpec.key

I don't think this is strictly necessary for all plugins, but something to keep in mind if we need to share state between plugins eventually.